### PR TITLE
add zh translation /docs/reference/setup-tools/kubeadm/generated/kube…

### PR DESCRIPTION
**Pull request by tanjunchen**

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_renew.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+* [kubeadm alpha certs renew controller-manager.conf](kubeadm_alpha_certs_renew_controller-manager.conf.md)	 - Renew the certificate embedded in the kubeconfig file for the controller manager to use
+* [kubeadm alpha certs renew etcd-healthcheck-client](kubeadm_alpha_certs_renew_etcd-healthcheck-client.md)	 - Renew the certificate for liveness probes to healthcheck etcd
+* [kubeadm alpha certs renew etcd-peer](kubeadm_alpha_certs_renew_etcd-peer.md)	 - Renew the certificate for etcd nodes to communicate with each other
+* [kubeadm alpha certs renew etcd-server](kubeadm_alpha_certs_renew_etcd-server.md)	 - Renew the certificate for serving etcd
+* [kubeadm alpha certs renew front-proxy-client](kubeadm_alpha_certs_renew_front-proxy-client.md)	 - Renew the certificate for the front proxy client
+* [kubeadm alpha certs renew scheduler.conf](kubeadm_alpha_certs_renew_scheduler.conf.md)	 - Renew the certificate embedded in the kubeconfig file for the scheduler manager to use -->
+* [kubeadm alpha 证书](kubeadm_alpha_certs.md)	 - 处理 kubernetes 证书相关的命令
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_renew.md]: need update

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_renew.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
+
+<!-- ### Synopsis -->
+### 简要
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_renew.md]:
```suggestion
### 概述
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_renew.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    <col span="1" style="width: 10px;" />
+    <col span="1" />
+  </colgroup>
+  <tbody>
+
+    <tr>
+      <td colspan="2">-h, --help</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">help for renew</td>
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_renew.md]:
```suggestion
      <td></td><td style="line-height: 130%; word-wrap: break-word;">renew 帮助命令</td>
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_renew.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+    <col span="1" style="width: 10px;" />
+    <col span="1" />
+  </colgroup>
+  <tbody>
+
+    <tr>
+      <td colspan="2">-h, --help</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;"> renew 帮助</td>
```
Comment (xichengliudui) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_renew.md]: 同上

**Comment by tanjunchen** [comment]
@xichengliudui  Thanks

**Pull request merged**